frontend/src/global_styles/openproject/_primer-adjustments.sass

```diff
@@ -64,3 +64,12 @@ page-header
 .Overlay
   &-body_autocomplete_height
     min-height: 300px
+
+// For some specific cases like progress form in small screens,
+// we need to change the direction of elements to column,
+// therefore elements are placed below each other
+.horizontal-form-group
```

Comment:
The sense of this class is to align the group vertically on mobile. I'd like that to be reflected in the class name. Further, please try to stick to our BEM convention, e.g `.Form_mobile-vertical` or something similar.